server/services/cohorts.js

```diff
@@ -75,9 +76,22 @@ const getPSICohorts = async (psiID) =>
           cohort_id: 'id',
         },
       },
+      participantStatusJoin: {
+        type: 'LEFT OUTER',
+        relation: collections.PARTICIPANTS_STATUS,
+        on: {
+          participant_id: 'participants.participant_id',
+        },
+      },
     })
     .find({
       psi_id: psiID,
+      // Additional condition to filter out the participants who don't have status assigned to them
```

Comment:
This comment reiterates what the code does but not why, we should aim to describe the intent of the code with comments

So acknowledging this as a safeguard against potential bad data and even referring to the ticket/GH discussion could offer more insight to a future reader